sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/FormRecognizerException.java

```diff
@@ -25,14 +27,15 @@
     public FormRecognizerException(final String message, final List<ErrorInformation> errorInformationList) {
         super(message);
         StringBuilder errorInformationStringBuilder = new StringBuilder().append(message);
-        if (errorInformationList.size() > 0) {
+        if (!errorInformationList.isEmpty()) {
```

Comment:
Would it make sense to use `!CoreUtils.isNullOrEmpty(errorInformationList)` , that will also check for null ?